libs/blocks/mnemonic-list/mnemonic-list.js

```diff
@@ -17,7 +17,13 @@ export const decorateMnemonicList = (container) => {
     if (title) product.appendChild(title);
     productList.appendChild(product);
     paragraph.replaceWith(productList);
+    paragraph.remove();
```

Comment:
Paragraph is already replaced by `productList`, so it should no longer be attached to the DOM, what purpose does this extra statement serve?

Reply:
good catch, removed.